src/lavinmq/amqp/queue/queue.cr

```diff
@@ -155,6 +155,9 @@ module LavinMQ::AMQP
 
     private def make_data_dir : String
       data_dir = File.join(@vhost.data_dir, Digest::SHA1.hexdigest @name)
+      unless durable?
+        data_dir = File.join(@vhost.data_dir, "transient", Digest::SHA1.hexdigest @name)
+      end
```

Comment:
minor: this way we don't do unnecessary assigns.. but since this is done once at init its really no big deal :)
 
```suggestion
      if durable?
        data_dir = File.join(@vhost.data_dir, Digest::SHA1.hexdigest @name)
      else
        data_dir = File.join(@vhost.data_dir, "transient", Digest::SHA1.hexdigest @name)
      end
```

Reply:
or:
```suggestion
      data_dir = if durable?
        File.join(@vhost.data_dir, Digest::SHA1.hexdigest @name)
      else
        File.join(@vhost.data_dir, "transient", Digest::SHA1.hexdigest @name)
      end
```